### PR TITLE
docs(readme file ): starting readme file

### PR DESCRIPTION
**Pull request by ilan-pinto**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -7,5 +7,8 @@
         "out": true // set this to false to include "out" folder in search results
     },
     // Turn off tsc task auto detection since we have the necessary tasks as npm scripts
-    "typescript.tsc.autoDetect": "off"
+    "typescript.tsc.autoDetect": "off",
+    "cSpell.words": [
```
Comment (TomerFi) [inline .vscode/settings.json:11]: I don't think this is required,
it's a misspelling of the word "contributors".

**Review by TomerFi — CHANGES_REQUESTED** [review]
We don't need _images/ocm-new-project-2.gif_,
it's an old version of _images/new-project.gif_.

We also don't need the empty file _CONTRIBUTING.md_,
we already have a _.github/CONTRIBUTING.md_,
we can move it to the root if you think it's better.

File: README.md [hunk 2]
```diff
@@ -1,71 +1,45 @@
-# ocm-vscode-extension README
+# OCM VScode Extension README
 
-This is the README for your extension "ocm-vscode-extension". After writing up a brief description, we recommend including the following sections.
+ "ocm-vs-extension" helps creating OCM cluster applications.
```
Comment (TomerFi) [inline README.md:3]: this should be "ocm-vscode-extension".

File: README.md [hunk 3]
```diff
@@ -1,71 +1,45 @@
...
+## Change Log
 
-**Note:** You can author your README using Visual Studio Code.  Here are some useful editor keyboard shortcuts:
+full change log can bd found [here](CHANGELOG.md) 
 
-* Split the editor (`Cmd+\` on macOS or `Ctrl+\` on Windows and Linux)
-* Toggle preview (`Shift+CMD+V` on macOS or `Shift+Ctrl+V` on Windows and Linux)
-* Press `Ctrl+Space` (Windows, Linux) or `Cmd+Space` (macOS) to see a list of Markdown snippets
+## Contributing 
+See our [Contributing Document](CONTRIBUTING.md) for more information
```
Comment (TomerFi) [inline README.md:41]: this should be (.github/CONTRIBUTING.md)

File: README.md [hunk 4]
```diff
@@ -1,71 +1,45 @@
...
+###### Release Date: 01/04/2022 
 
-Ensure that you've read through the extensions guidelines and follow the best practices for creating your extension.
+- Loading OCM crd snippets from the command pallette OCM  
+- Create a sample Git-type Application Lifecycle Project
 
-* [Extension Guidelines](https://code.visualstudio.com/api/references/extension-guidelines)
 
-## Working with Markdown
+## Change Log
```
Comment (TomerFi) [inline README.md:36]: I'm not sure we need this section (and the CHANGELOG.md file) at all.
If we're going to leverage GitHub releases, they have a great feature for auto-generating the release notes and the compare link.
maintaining a CHANGELOG.md file and a designated section in the README.md file seems redundant.

File: README.md [hunk 5]
```diff
@@ -1,71 +1,45 @@
...
-
-Fixed issue #.
-
-### 1.1.0
-
-Added features X, Y, and Z.
-
------------------------------------------------------------------------------------------------------------
-## Following extension guidelines
+### 1.0.0
```
Comment (TomerFi) [inline README.md:29]: I'm not sure we need this section at all.
If we're going to leverage GitHub releases,
they have a great feature for auto-generating the release notes.
maintaining a designated section in the README.md file seems redundant.

**Review by TomerFi — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (ilan-pinto) [reply README.md:36]: OK, I am removing it.

File: .vscode/settings.json (hunk 1 above)
Comment (TomerFi) [reply .vscode/settings.json:11]: @ilan-pinto you forgot to resolve this in the latest commit.
😉

Comment (TomerFi) [reply .vscode/settings.json:11]: btw
I haven't heard of _cspell_,
looks awesome!
😄

**Review by TomerFi — APPROVED** [review]

**Pull request merged**